warehouse/forklift/legacy.py

```diff
@@ -143,8 +143,8 @@ def namespace_stdlib_list(module_list):
     "11",
 }
 
-# manylinux pep600 is a little more complicated:
-_manylinux_platform_re = re.compile(r"manylinux_(\d+)_(\d+)_(?P<arch>.*)")
+# manylinux pep600 and musllinux pep656 are a little more complicated:
+_manylinux_platform_re = re.compile(r"m(any|usl)linux_(\d+)_(\d+)_(?P<arch>.*)")
```

Comment:
According to [auditwheel policy file](https://github.com/pypa/auditwheel/blob/master/auditwheel/policy/musllinux-policy.json), musllinux doesn't support `ppc64` architecture.

Reply:
Thanks, this should be handled now.